src/main/resources/db/migration/V1__init_iam_database.sql

```diff
@@ -118,8 +118,10 @@ CREATE UNIQUE INDEX user_policies_idx_principal_policy ON user_policies(principa
 
 CREATE TABLE master_keys (
   id uuid DEFAULT gen_random_uuid() PRIMARY KEY,
-  private_key BYTEA NOT NULL,
-  public_key BYTEA NOT NULL,
+  private_key_pem BYTEA NOT NULL,
+  private_key_der BYTEA NOT NULL,
+  public_key_pem BYTEA NOT NULL,
+  public_key_der BYTEA NOT NULL,
   status VARCHAR(10) NOT NULL, -- SIGNING, VERIFYING, EXPIRED
```

Comment:
In IAM backend, We must ideally authorize requests which are signed using an unexpired master key. But, that check seems to be missing which is a bug that needs to be fixed. Also, the same behavior needs to incorporated into clients as well. Will returning a non-2XX code from `/public_keys/<kid>` request if kid has expired help the clients?